CHANGELOG.rst

```diff
@@ -4,65 +4,91 @@ Cisco Ios Collection Release Notes
+
+Bugfixes
+--------
+
+- Add support for autoconfig and dhcp keywords for IPv6 addresses in l3_interfaces (https://github.com/ansible-collections/cisco.ios/pull/269).
+- Reordering names of interface for proper value assignment
+- fixes Serial interface configuration for l3_interfaces module and Unit Test cases added.
+- fixes banner module with new attribute introduced
+- fixes soft_reconfiguration and prefix_list command formation.
+
 v2.2.0
 ======
 
 Minor Changes
 -------------
-- Add ios_logging_global module. (https://github.com/ansible-collections/cisco.ios/pull/306).
-- Add Prefix list resource module (https://github.com/ansible-collections/cisco.ios/pull/308).
+
+- Add ios_logging_global module.
+- IOS Prefix list resource module.
 
 Bugfixes
 --------
-- To fix IOS vlans RM where traceback was thrown if show vlan wasn't supported on the device and also fix replace and overridden state behaviour.
+
+- Fix IOS bgp global RM tracback while there's no bestpath/nopeerup_delay configured.
 - Fix logging commands for v12 versions (https://github.com/ansible-collections/cisco.ios/issues/207).
+- To fix IOS vlans RM where traceback was thrown if show vlan wasn't supported on the device and also fix replace and overridden state behaviour.
+- To fix Spelling glitch.
 - To fix ios acls overridden and replaced state of their inconsistent behaviour (https://github.com/ansible-collections/cisco.ios/issues/250).
-- Fix IOS bgp global RM tracback while there's no bestpath/nopeerup_delay configured.
 - To fix ios_bgp_address_family neighbor next_hop_self param (https://github.com/ansible-collections/cisco.ios/issues/319).
-- To fix Spelling glitch.
 
 New Modules
 -----------
 
-- ios_logging_global - This module manages the logging attributes of Cisco IOS network devices.
-- ios_prefix_lists - This module configures and manages the attributes of prefix list on Cisco IOS.
+- ios_logging_global - Logging resource module.
+- ios_prefix_lists - Prefix Lists resource module.
 
 v2.1.0
 ======
 
 Minor Changes
 -------------
-- Add support for ansible_network_resources key allows to fetch the available resources for a platform (https://github.com/ansible-collections/cisco.ios/pull/292).
+
 - Add ios_route_maps Resource Module (https://github.com/ansible-collections/cisco.ios/pull/297).
+- Add support for ansible_network_resources key allows to fetch the available resources for a platform (https://github.com/ansible-collections/cisco.ios/pull/292).
 
 Security Fixes
 --------------
+
 - To fix Cisco IOS no log issue and add ignore txt for 2.12 (https://github.com/ansible-collections/cisco.ios/pull/304).
 
 Bugfixes
 --------
-- To fix the wrong arg being passed in acls template function (https://github.com/ansible-collections/cisco.ios/pull/305).
 
-New Modules
------------
-
-- ios_route_maps - Manage the attributes of Route maps on Cisco IOS.
```

Comment:
why this is removed?

Reply:
manually added back, thanks @ashwini-mhatre